### PR TITLE
Document manual release

### PR DESCRIPTION
**Pull request by jotak**

### CODE REVIEW

**Comment by jotak** [comment]
These steps can be used as a basis for the prow releasing job that we'll need to create for all components

File: Makefile [hunk 1]
```diff
@@ -55,13 +55,13 @@ push:
 
 .PHONY: ovnk-deploy
 ovnk-deploy:
-	sed -e 's~quay\.io/netobserv/goflow-kube:dev~$(IMAGE):$(VERSION)~' ./examples/goflow-kube.yaml | kubectl apply -f - && \
+	sed -e 's~quay\.io/netobserv/goflow2-kube:main~$(IMAGE):$(VERSION)~' ./examples/goflow-kube.yaml | kubectl apply -f - && \
```
Comment (OlivierCazade) [inline Makefile:58]: Should the prow release process be also responsible for creating this image tags?
Or should we have different release process?

**Review by OlivierCazade — APPROVED** [review]
LGTM, thanks!

File: Makefile (hunk 1 above)
Comment (jotak) [reply Makefile:58]: Not sure to understand, this target is intended for local dev/test purpose where you can use your own image/tag, so I would leave prow out of that.

Or do you suggest that prow updates the sample YAML to make it point to the latest release?

IMO we can keep prow just for the release process and keep also the `main` tag that is more useful for dev/testing and is still created from a quay trigger

**Review by mariomac — APPROVED** [review]

**Pull request merged**